packages/SwingSet/test/test-vpid-kernel.js

```diff
@@ -575,7 +558,7 @@ async function doTest4567(t, which, mode) {
   };
   onDispatchCallback = function odc1(d) {
     t.deepEqual(d, resolutionOf(p1VatA, mode, targetsA));
-    t.is(inCList(kernel, vatA, p1kernel, p1VatA), !expectRetirement);
+    t.is(inCList(kernel, vatA, p1kernel, p1VatA), expectRetirement);
```

Comment:
This seems weird, and I suspect that `expectRetirement` is no longer being exercised correctly. If we expect the item to be retired, why would we assert that it's still in the clist?

Reply:
Oh, right, this is related to my recommendation that we remove the clist entry *before* giving control to the vat. Since the current branch is not doing that, this callback (which runs while the vat has control) *will* observe the clist entry to be present (independent of `expectRetirement`). Then, a few lines later, after the dispatch has completed, we test the clist again, and expect the entry to be removed.

To match the behavior of the code, this line should change to use `true` instead of `expectRetirement`.

If we can remove the clist entry before giving control to the vat, then this test should change to `false` instead of `expectRetirement`.